test/server-side-rendering/index.ts

```diff
@@ -145,7 +142,7 @@ describe('ssr', () => {
 
 	// duplicate client-side tests, as far as possible
 	runRuntimeSamples('runtime');
-	runRuntimeSamples('runtime-puppeteer');
+	// runRuntimeSamples('runtime-puppeteer');
```

Comment:
should this be uncommented?

Reply:
hmm.. i've reverted this change. i think the comment was a mistake.